kitsune/search/tasks.py

```diff
@@ -121,7 +121,7 @@ def index_chunk_task(write_index, batch_id, chunk):
 
     except Exception:
         rec.text = u'{0}: Errored out {1} {2}'.format(
-            rec.text, sys.exc_type, sys.exc_value)
+            rec.text, sys.exc_type, sys.exc_value)[:255]  # Truncate at 255 chars.
```

Comment:
The note says that some exceptions aren't pickleable. Pretty sure the problem is that this can be executing in celery-land, so it's important to throw pickleable things otherwise celery can't pickle it as a return value and then things get hosed.